### DISCUSSION TITLE
Enhanced Workflow | Public variables creation from Project Template

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#435] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

12.2.0

### Description

Hi guys,
Is it possible to set some public variables to be automatically created in new projects, starting from a Project Template, **using the Enhanced Workflow**?

e.g. I want to create a new project with the Enhanced Workflow and to automatically have the `LOG_LEVEL` public variable in my project.

Without using the Enhanced Workflow is possible by creating the `variables.env` file inside the template repository, I tried the same way with the Enhanced but I failed.

Is there a way to do that?

### Actual Outcome

I don't have public variables in my new project with the Enhanced Workflow

### Expected Outcome

I want some predefined public variables in my new project with the Enhanced Workflow

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, project creation for the Enhanced Workflow supports all the features for project template already in place. 

This means that you can still define an `api-console-config.json` file within the contents of your Project Template and it will be imported as initial configuration for your newly created project. Therefore you can easily define any resource you want, including public variables

> [!NOTE]  
Of course since the Enhanced Workflow does not require the file to exist in the actual project repository you won't find the file in the created project but everything in the design section will be properly set up)

**Reply by frengooo** [reply]
Hi @fredmaggiowski,
thanks for the reply, I'll try it.

Since the file is not needed, I could delete it after the project creation, right?

**Reply by fredmaggiowski** [reply]
>Since the file is not needed, I could delete it after the project creation, right?

In the created project the file will be automatically omitted so there is not need to delete it them later
